matrix/rules.py

```diff
@@ -147,7 +147,7 @@ def load_suite(self, filelike):
         subscription = None
         period = None
         if rule.has("periodic"):
-            period = rule.select_one("periodic").statement[0]
+            period = int(rule.select_one("periodic").statement[0])
```

Comment:
You know, I had a trailing comma that I only noticed due to the explicit error after I added this, so it probably will be coerced now that I've removed that.  I'll back this out.